app/modules/listings/components/listing_element.svelte

```diff
@@ -23,70 +29,130 @@
   } else if (image?.url) {
     imageUrl = image.url
   }
+
+  function toggleShowMode () {
+    showElementModal = !showElementModal
+    autocompleteFlash = null
+    if (showElementModal) {
+      const options = { metadata: getElementMetadata(listing, element), preventScrollTop: true }
```

Comment:
This line crashes when after editing the comment I close the modal and try to re-open it:

```
entity is undefined
getElementTitle@webpack-internal:///./app/modules/listings/lib/listings.ts:212:13
async*getElementMetadata@webpack-internal:///./app/modules/listings/lib/listings.ts:184:18
toggleShowMode@webpack-internal:///./app/modules/listings/components/listing_element.svelte:688:90
```

Reply:
fixed in ce6f68597